extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -231,8 +238,9 @@ async function getCommandsInPath(env: { [key: string]: string | undefined } = pr
 			const files = await vscode.workspace.fs.readDirectory(vscode.Uri.file(path));
 
 			for (const [file, fileType] of files) {
-				if (fileType !== vscode.FileType.Unknown && fileType !== vscode.FileType.Directory && await isExecutable(path + pathSeparator + file)) {
-					executables.add(file);
+				if (!labels.has(file) && fileType !== vscode.FileType.Unknown && fileType !== vscode.FileType.Directory && await isExecutable(path + pathSeparator + file)) {
+					executables.add({ label: file, path: path + pathSeparator + file });
```

Comment:
Is this the right format? We want it to use `.fsPath` to make sure the presentation is the same as with directories. We can do that via `vscode.Uri.joinPath(path, file)`. You can add a helper to the extension similar to this one:

https://github.com/microsoft/vscode/blob/09d6f4b00374c6aa5d03775cee589314d0340145/src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts#L354-L365

We also want that drive letter tweak.